sys/shell/cmds/vfs.c

```diff
@@ -720,6 +720,91 @@ static int _vfs_handler(int argc, char **argv)
 
 SHELL_COMMAND(vfs, "virtual file system operations", _vfs_handler);
 
+#if MODULE_SHELL_CMD_GENFILE
+static char _get_char(unsigned i)
+{
+    i %= 2 * (1 + 'z' - 'a') + 10;
+
+    if (i < 10) {
+        return '0' + i;
+    }
+    i -= 10;
+
+    if (i <= 'z' - 'a') {
+        return 'a' + i;
+    }
+    i -= 1 + 'z' - 'a';
+
+    return 'A' + i;
+}
+
+static void _write_block(int fd, unsigned bs, unsigned i)
+{
+    char block[bs];
+    char *buf = block;
+
+    buf += snprintf(buf, bs, "|%03u|", i);
+
+    memset(buf, _get_char(i), &block[bs] - buf);
+    block[bs - 1] = '\n';
+
+    vfs_write(fd, block, bs);
+}
+
+static int _vfs_genfile_cmd(int argc, char **argv)
+{
+    unsigned blocksize = 64;
+    unsigned blocks = 32;
+    int fd = STDOUT_FILENO;
+
+    const char *cmdname = argv[0];
+    while (argc > 1 && argv[1][0] == '-') {
```

Comment:
The argument parsing is insufficient. Cases like `genfile lkjfsalkjas` are treated as legit.